.github/workflows/crowdin-upload.yml

```diff
@@ -45,12 +45,12 @@ jobs:
       - name: Crowdin sync
         uses: crowdin/github-action@v2
         with:
-          upload_sources: true
+          upload_sources: ${{ vars.CROWDIN_UPLOAD_SOURCES || false }}
```

Comment:
We always want upload sources `true`.

I think you meant to add this to `upload_translations`.

Is vars enough? I was expecting something like this: 

[Github Actions Inputs](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs)

Reply:
Ack! Thanks!

Variables should be enough. Dispatch inputs looks fancier but a little harder to set up 🤔 Is it super important they are set on the dispatch? I don't think we'll be toggling very often.

Reply:
@Duncan-Brain it looks really promising

![Screenshot 2024-07-29 at 11 18 16 AM](https://github.com/user-attachments/assets/ed22eab2-a75a-4577-8173-b369ff7c9566)

... but I'm a little bit worried about this wording in your [linked docs here](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatch):

> This trigger only receives events when the workflow file is on the default branch.

Reply:
I just had never heard of variables context and was trying to figure where they came from.... I see them now in the secrets area! Interesting..  I am good with it! 

Pros for workflow inputs:
- Github does mention Variables are in beta and can change at any time.
- Inputs are workflow specific and accessible without navigating through github settings to update/revert
- Variable seem best suited for across multiple actions files

Cons for workflow inputs
- might be easier for someone else to make a mistake

Reply:
Oh you beat me to it, yeah that is a concerning line ! -- defaults to false so risk should be low.

Totally up to you!

Reply:
Despite the warning, I just ran it on the unmerged feature branch with "upload translations" checked and it seemed to upload them. Dunno if that's a good thing or not 😂

I'm interested in this option; we can always have variables in reserve as our fallback if it doesn't work, they are so easy to use.